hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -959,7 +974,7 @@
       uvm_test_seq: chip_sw_keymgr_key_derivation_vseq
       sw_images: ["//sw/device/tests/sim_dv:keymgr_key_derivation_test:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
-      run_opts: ["+lc_at_prod=1", "+sw_test_timeout_ns=20_000_000"]
```

Comment:
I didn't check to see if `lc_at_prod` enables other things besides setting the LC state value.

Reply:
Yeah @weicaiyang mentioned that this may be used somewhere else in the TB. I reverted the change. We can clean this up later.